src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt

```diff
@@ -0,0 +1,43 @@
+package hs.kr.equus.user.global.utils.pass
+
+import hs.kr.equus.user.domain.auth.exception.InvalidOkCertConnectException
+import kcb.module.v3.OkCert
+import kcb.module.v3.exception.OkCertException
+import kcb.org.json.JSONObject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class PassUtil {
+
+    companion object {
+
+        private val okCert = OkCert()
+
+        private val TARGET = "PROD"
+
+        private val SVC_NAME = "IDS_HS_POPUP_RESULT"
+
+        private val MODEL_TOKEN = "MDL_TKN"
+    }
+
+    @Value("{pass.cp-cd}")
+    private lateinit var CP_CD: String
+
+    @Value("{pass.license}")
+    private lateinit var LICENSE: String
```

Comment:
🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case